views/results/_form.html

```diff
@@ -1,13 +1,15 @@
 <div class="card card-upload">
-    <h3 class="card__title">Upload your keywords here</h3>
-    <p class="card__subtitle">
-        {{ render_icon "info" "icon--sm" }}
-        Only CSV file allowed, check
-        <a href="/files/sample.csv"> this sample file </a>
-        for the file format
-    </p>
-    <form class="form form--inline form-upload" action="/results" method="post" enctype="multipart/form-data">
-        <input id="file" name="file" type="file" accept="text/csv" class="form__input" required>
-        <input type="submit" value="Submit" class="form__submit-button"/>
-    </form>
+    <div class="card-body">
+        <h3 class="card-title">Upload your keywords here</h3>
+        <p class="card-subtitle">
```

Comment:
Should it be `card-body__subtitle`?

Reply:
@gutakk Regarding [this answer](https://github.com/carryall/go-google-scraper-challenge/pull/73#discussion_r645513182) so I want to keep it align with the title